plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -463,7 +463,7 @@ func ClusterRoles() []rbacv1.ClusterRole {
 	// node-proxier role is used by kube-proxy.
 	nodeProxierRules := []rbacv1.PolicyRule{
 		rbacv1helpers.NewRule("list", "watch").Groups(legacyGroup).Resources("services", "endpoints").RuleOrDie(),
-		rbacv1helpers.NewRule("get").Groups(legacyGroup).Resources("nodes").RuleOrDie(),
+		rbacv1helpers.NewRule("get", "list", "watch").Groups(legacyGroup).Resources("nodes").RuleOrDie(),
```

Comment:
I'm not sure this is the case anymore but I was under the impression that we generally only make additive changes to bootstrap policies since removing rules can get ugly on downgrade.

Reply:
I don't think it has needed "get" for a long time :)